www/src/Addons/Wii.tsx

```diff
@@ -43,91 +47,280 @@ export const wiiState = {
+                        groupClassName="col-sm-3 mb-3"
+                        value={values.wiiExtensionSDAPin}
+                        error={errors.wiiExtensionSDAPin}
+                        isInvalid={errors.wiiExtensionSDAPin}
+                        onChange={handleChange}
+                        min={-1}
+                        max={29}
+                    />
+                    <FormControl
+                        type="number"
+                        label={t('WiiAddon:scl-pin-label')}
+                        name="wiiExtensionSCLPin"
+                        className="form-select-sm"
+                        groupClassName="col-sm-3 mb-3"
+                        value={values.wiiExtensionSCLPin}
+                        error={errors.wiiExtensionSCLPin}
+                        isInvalid={errors.wiiExtensionSCLPin}
+                        onChange={handleChange}
+                        min={-1}
+                        max={29}
+                    />
+                    <FormSelect
+                        label={t('WiiAddon:block-label')}
+                        name="wiiExtensionBlock"
+                        className="form-select-sm"
+                        groupClassName="col-sm-3 mb-3"
+                        value={values.wiiExtensionBlock}
+                        error={errors.wiiExtensionBlock}
+                        isInvalid={errors.wiiExtensionBlock}
+                        onChange={handleChange}
+                    >
+                        {I2C_BLOCKS.map((o, i) => (
+                            <option
+                                key={`wiiExtensionI2cBlock-option-${i}`}
+                                value={o.value}
+                            >
+                                {o.label}
+                            </option>
+                        ))}
+                    </FormSelect>
+                    <FormControl
+                        label={t('WiiAddon:speed-label')}
+                        name="wiiExtensionSpeed"
+                        className="form-control-sm"
+                        groupClassName="col-sm-3 mb-3"
+                        value={values.wiiExtensionSpeed}
+                        error={errors.wiiExtensionSpeed}
+                        isInvalid={errors.wiiExtensionSpeed}
+                        onChange={handleChange}
+                        min={100000}
+                    />
+                </Row>
+                <Row className="mb-3">
+                    <Tabs
+                        defaultActiveKey={`wii${WII_EXTENSION_CONTROLS[0].id}Config`}
+                        id="wiiControllerConfig"
+                        className="mb-3"
+                        fill
+                    >
+                        {WII_EXTENSION_CONTROLS.map((controlObj, controlID) => (
+                            <Tab eventKey={`wii${controlObj.id}Config`} title={t(`WiiAddon:controller-${controlObj.id.toLowerCase()}`)}>
```

Comment:
We are getting some warnings in the console:
`Warning: Each child in a list should have a unique "key" prop.` 

We have these in other places but if possible add a unique `key` prop when looping some components

Reply:
I would need to break down react-bootstrap's Tab to a NavItem and such to be able to apply a key here.  Would take some doing, but that seems to be what's going on here.

![image](https://github.com/OpenStickCommunity/GP2040-CE/assets/1214033/31c851a6-4f31-48b7-923f-49c1b76d4508)

Reply:
Fixed this with a fresh night of sleep. Sometimes it's not very clear where the warnings come from in the console.